packages/financial-templates-lib/src/price-feed/DefaultPriceFeedConfigs.js

```diff
@@ -384,9 +384,8 @@ const defaultConfigs = {
   CNYUSD: {
     type: "tradermade",
     pair: "CNYUSD",
-    minuteLookback: 7200, // interval=minute allowed lookback up to 2 days
```

Comment:
We should be setting defaults for `minuteLookback` and `hourlyLookback` within the `TraderMadePriceFeed`. The `lookback` will be automatically set by the client `liquidator`, `disputer`, or `monitor` bot but we can't really assume the window of time for which we want to query the timeseries data.

So we'll basically just ask for more data than we neccessarily need to lookback through.